ccextender.yaml

```diff
@@ -1,44 +1,316 @@
-      - oss
+      prompt: "Will your project be [1] a service or [2] a library?"
+    service:
+      - service
+    library:
+      - library
+  source-python-service:
+    query:
+      prompt: "Will this be [1] an open source or [2] a private source project?"
+      include-if:
+        - python
+        - service
+    python-open:
+      - python-open-service
+    python-private:
+      - python-private-service
+  source-python-library:
+    query:
+      prompt: "Will this be [1] an open source or [2] a private source project?"
+      include-if:
+        - python
+        - library
+    python-open:
+      - python-open-library
+    python-private:
+      - python-private-library
+  source-golang-service:
+    query:
+      prompt: "Will this be [1] an open source or [2] a private source project?"
+      include-if:
+        - golang
+        - service
+    golang-open:
+      - golang-open-service
+    golang-private:
+      - golang-private-service
+  source-golang-library:
+    query:
+      prompt: "Will this be [1] an open source or [2] a private source project?"
+      include-if:
+        - golang
+        - library
+    golang-open:
+      - golang-open-library
+    golang-private:
+      - golang-private-library
+  io:
+    query:
+      prompt: "Will you need [1] both an input stream and output stream, [2] just an input stream, [3] just an output stream, or [0] no message streams?"
+      exclude-if:
+        - golang-open
+        - python-open
+        - library
+    both-io:
+      - input
+      - output
+      - streamer-manager
+    input:
+      - input
+      - streamer-manager
+    output:
+      - output
+      - streamer-manager
 change-packs:
-  docker:
-    template-go-docker:
-  go-service:
-    template-go-layout:
-    template-go-lint:
-  oss:
-    template-oss:
+  golang-base:
+    template-base:
+    template-go-base:
+  python-base:
+    template-base:
+    template-python-base:
+  library:
+  service:
+  golang-open-library:
+    # Currently nothing needed here
+  python-open-library:
+    # Currently nothing needed here
+  golang-private-library:
+    template-go-private-library:
+  python-private-library:
+    # Currently nothing needed here
+  python-open-service:
+    template-makefile:
+        sdcli_lang: python
+    template-python-open-service:
+    template-docker:
+  python-private-service:
+    template-makefile:
+        sdcli_lang: python
+    template-python-private-service:
+  golang-open-service:
+    template-go-open-service:
+    template-docker:
+    template-makefile:
+      sdcli_lang: go
+  golang-private-service:
+    template-go-private-service:
+    template-makefile:
+      sdcli_lang: go
+  input:
+    template-inbound:
+    template-micros:
+      service_descriptor: "
+                           \ benthos-input:\n
+                           \   image: ${BENTHOS_INPUT_ARTIFACT_NAME} #docker.atl-paas.net/sox/asecurityteam/sec-nexpose-asset-producer-benthos-input\n
```

Comment:
Question: I'm confused at why `nexpose-asset-producer` is here. Should these insert the service name given during template generation instead? Same for the `awsconfig-filterd` that's specified below

Reply:
Oops! Good catch. I'd seen those earlier, and thought I'd already taken them out. Sorry about that, I'll upload a fix for it soon.